2024.5/metagenome/passed/seed-environment-conda.yml

```
@@ -4,17 +4,18 @@ channels:
 - bioconda
 - defaults
 dependencies:
-- biom-format=2.1.14
+- biom-format=2.1.15
 - bowtie2=2.5.1
-- h5py=3.8.0
+- h5py=3.11.0
 - hdf5=1.14.0
```

```suggestion
- hdf5=1.14.3
```
Looks like this was change was used in the amplicon distro and is related to the solve error

---

i think we've got at least a few things that need to be updated in order to get a solve. i'm going to do a local install and work through all of the external deps for metagenome-specific plugins to see where that lands us.